contrib/kube-prometheus/Makefile

```diff
@@ -18,6 +18,7 @@ generate-in-docker: ../../hack/jsonnet-docker-image
 	--rm \
 	-u=$(shell id -u $(USER)):$(shell id -g $(USER)) \
 	-v $(shell dirname $(dir $(abspath $(dir $$PWD)))):/go/src/github.com/coreos/prometheus-operator/ \
+	-v $(shell go env GOCACHE):/.cache/go-build \
```

Comment:
As far as I know, go cache invalidation only happens based on golang code changes. If we change the environment itself, tests would not be rerun.

Thereby I find this a bit risky compared to the time saved through it. What do you think?